### ISSUE TITLE
grid size is fixed when mapOnFly = false

### ISSUE THREAD

**Issue by sonjageorgievska** [DeepRank/deeprank#80]
Li told me to note down this, so that everyone knows: if the training uses already mapped features in 3D,  then the network will use the resolution as it is in the hdf5 files.

**Comment by NicoRenaud** [comment]
Yes it should be better documented then :) Or do you suggest to change that somehow ?

**Comment by CunliangGeng** [comment]
@LilySnow Should put this info to documentation.
